apis/src/components/organisms/dropdowns/locale.rs

```diff
@@ -0,0 +1,46 @@
+use crate::components::layouts::base_layout::{COMMON_LINK_STYLE, DROPDOWN_BUTTON_STYLE};
+use crate::components::molecules::hamburger::Hamburger;
+use crate::i18n::*;
+use leptos::*;
+
+const DROPDOWN_MENU_STYLE: &str = "flex flex-col items-stretch absolute bg-even-light dark:bg-gray-950 text-black border border-gray-300 rounded-md left-34 p-2";
+
+const ALL_LOCALES: [Locale; 12] = [
```

Comment:
We need to remove the ones that are barely translated. Otherwise we can deploy this as is I think